app/src/main/java/com/example/giftly/FontSizeScreen.java

```diff
@@ -90,13 +93,12 @@ public void onClick(View view) {
             }
         });
 
-        //Enable the back-button
+        //The action bar will display the back button in the header
         ActionBar actionBar = getSupportActionBar();
         actionBar.setDisplayHomeAsUpEnabled(true);
-
-
     }
 
+    //This will loop through Button and EditText fields in the xml files to change the text sizes
```

Comment:
Can we flip the dependency here and have the UI call system preferences to set edit text so you don't have to iterate everything now?

Reply:
was not able to find another solution that could iterate through xml Buttons and EditText fields

Reply:
Got it, we'll put it on the backburner and try later